crates/iota-swarm/src/memory/swarm.rs

```diff
@@ -286,6 +290,19 @@ impl<R> SwarmBuilder<R> {
         self
     }
 
+    pub fn with_fullnode_grpc_api_config(mut self, config: iota_grpc_api::Config) -> Self {
+        self.fullnode_grpc_api_config = Some(config);
+        self
+    }
+
+    pub fn with_fullnode_grpc_api_address(mut self, addr: SocketAddr) -> Self {
+        self.fullnode_grpc_api_config = Some(iota_grpc_api::Config {
```

Comment:
same here, it would overwrite existing configs. Maybe remove the `with_fullnode_grpc_api_address`.

Reply:
I modified it as the way @semenov-vladyslav suggested in [c09b0c1](https://github.com/iotaledger/iota/pull/7511/commits/c09b0c1f0cd4007b6567cf29b8e290db81294dd9).